js/interface.js

```diff
@@ -58,7 +59,16 @@ function getDataSources() {
   $dataSources = $('#data-sources > tbody');
 
   Fliplet.DataSources.get().then(function (dataSources) {
```

Comment:
The filter is to choose only menus.
But we can enhance it and check if null then send only null types.